aodndata/moorings/products_handler.py

```diff
@@ -43,25 +43,31 @@ def __init__(self, *args, **kwargs):
         self.allowed_extensions = ['.json_manifest']
         self.product_site_code = None
         self.product_variables = None
+        self.input_file_collection = None
+        self.input_file_variables = None
+        self.excluded_files = defaultdict(set)
```

Comment:
I don't see a reason to use defaultdict here since the dict is very simple - explicit is better than implicit.